### ISSUE TITLE
Flip Screen DIP Switch Not Working (All Games)

### ISSUE THREAD

**Issue by MJY71** [MiSTer-devel/Arcade-IremM62_MiSTer#6]
The Flip Screen DIP switch has no effect on any game, with the exception of Kid Niki - Radical Ninja and Youjyuden, where it seems to only shift the background layer side-to-side a small amount.

**Comment by SegaSnatcher** [comment]
Yeah, was about to post this same issue after trying Youjyuden and having no luck with flip mode.  This will be really useful for me since my monitor can only rotate clockwise.

**Comment by M-Walrus** [comment]
Just was about to report this for The Battle-Road, Horizon & Kung-Fu Master. Looks like it affects all games in the core.

**Comment by CMR779** [comment]
Is there still no fix for this?
